test/e2e-common.sh

```diff
@@ -30,6 +30,12 @@ which gcloud &> /dev/null || gcloud() { echo "[ignore-gcloud $*]" 1>&2; }
 
 # Eventing main config.
 readonly E2E_TEST_NAMESPACE="default"
+readonly CONTROL_PLANE_NAMESPACE="cloud-run-events"
```

Comment:
@chizhg these are the changes I'm talking about... But there is an IF not PROW for the others. I want to see how we can add this to PROW?

Reply:
The service account we use to run Prow jobs already has `roles/editor`, so if the tests are run by Prow, there is no need to create another service account.
The current logic you have written is correct.

Reply:
Sounds good! Thanks @chizhg